### ISSUE TITLE
Question about output ordering with replicates

### ISSUE THREAD

**Issue by borisz-panorama** [Xinglab/rmats-turbo#263]
When rmats.py is run in post mode, in the output files (such as SE.MATS.JCEC.txt), are the comma-seperated replicate read counts and PSI values for IJC_SAMPLE_1, IncLevel1, etc. all guaranteed to be in the same order as the bam files are provided in the --b1 and --b2 files? In other words does the first value in IJC_SAMPLE_1 correspond to the first sample in the --b1 file and  the first value in IJC_SAMPLE_2 correspond to the first sample in the --b2 file and so on?

I could not find this stated in the instructions.

**Comment by EricKutschera** [comment]
Yes the values in the output columns for sample 1 (IJC_SAMPLE_1, SJC_SAMPLE_1, IncLevel1) are guaranteed to be in the same order as --b1. The sample 2 output columns are in the same order as --b2
